tests/test_transform_pink_noise.py

```diff
@@ -74,3 +74,18 @@ def test_pink_noise(duration, sampling_rate, gain_db, snr_db):
     assert noise.shape == expected_noise.shape
     assert noise.dtype == expected_noise.dtype
     np.testing.assert_almost_equal(noise, expected_noise)
+
+
+@pytest.mark.parametrize(
+    "signal",
+    [
+        # Odd,
+        # compare https://github.com/audeering/auglib/issues/23
+        np.ones((1, 30045)),
+        # Even
+        np.ones((1, 200)),
+    ],
+)
+def test_pink_noise_odd_and_even_samples(signal):
+    transform = auglib.transform.PinkNoise()
+    transform(signal)
```

Comment:
Maybe this test could also check that the output signal length is the same as the input signal length

Reply:
Good idea, I extended the test.